src/AccessibilityInsights.SharedUx/Dialogs/StartUpModeControl.xaml.cs

```diff
@@ -86,7 +78,7 @@ public void UpdateHotkeyLabels()
         /// <param name="e"></param>
         private void btnExit_Click(object sender, RoutedEventArgs e)
         {
```

Comment:
The previous version is correct, no? If the "don't show" checkbox is checked, ShowWelcomeScreenOnLaunch is set to false.

Reply:
The checkbox's text has changed so that it is now a "do show" checkbox, so this needed to be changed as well

Reply:
Updated name